src/shogun/lib/tapkee/tapkee_shogun.cpp

```diff
@@ -1,8 +1,4 @@
-/*
- * This program is free software; you can redistribute it and/or modify
- * it under the terms of the GNU General Public License as published by
- * the Free Software Foundation; either version 3 of the License, or
- * (at your option) any later version.
+/* This software is distributed under BSD 3-clause license (see LICENSE file).
```

Comment:
Sure, like done here, it is a one-liner referencing to the LICENSE file, @karlnapf.